internal/keppel/manifest.go

```diff
@@ -24,158 +24,235 @@ import (
 
 	"github.com/sapcc/keppel/internal/models"
 
-	"github.com/docker/distribution"
-	v1 "github.com/opencontainers/image-spec/specs-go/v1"
-
-	// distribution.UnmarshalManifest() relies on the following packages
-	// registering their manifest schemas.
-	"github.com/docker/distribution/manifest/manifestlist"
-	"github.com/docker/distribution/manifest/ocischema"
-	"github.com/docker/distribution/manifest/schema2"
+	"github.com/containers/image/v5/manifest"
+	"github.com/containers/image/v5/types"
+	imageSpecs "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
Camel case is not commonly used for import names:
```suggestion
	imagespecs "github.com/opencontainers/image-spec/specs-go/v1" 
```